github_connector/models/abstract_github_model.py

```diff
@@ -0,0 +1,286 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+from urllib.request import urlopen
+import logging
+
+from odoo import _, api, exceptions, fields, models, tools
+from odoo.exceptions import UserError
+
+from .github import Github
+
+_logger = logging.getLogger(__name__)
+
+
+class AbstractGithubModel(models.AbstractModel):
+    """
+    This abstract model is used to share all features related to github model.
+    Note that some fields and function have to be defined in the inherited
+    model. (github_type for instance)
+    """
+
+    _name = 'abstract.github.model'
+    _github_type = None
```

Comment:
I get this warning:
`abstract.github.model has no _description `
so I think It's better that you add the parameter
`_description`